### PR TITLE
Fixes a case Adrian found where a tensor is frozen

### PR DESCRIPTION
**Pull request by lukas**
Fixed your bug @adrianbg

### CODE REVIEW

File: tests/test_torch.py [hunk 1]
```diff
@@ -219,7 +218,6 @@ def test_sequence_net():
         (97, 999)))
     output.backward(torch.zeros((97, 999)))
     graph = wandb.Graph.transform(graph)
-    pprint(graph)
```
Comment (adrianbg) [inline tests/test_torch.py]: Prints in tests are okay because they only actually get printed if the test fails.. unless you really think this is useless

Comment (lukas) [reply tests/test_torch.py]: its kind of random i guess.

File: wandb/wandb_torch.py [hunk 2]
```diff
@@ -74,8 +74,9 @@ def parameter_log_hook(module, input_, output):
         # This won't handle the case if the network changes
         if log_gradients:
             for name, parameter in module.named_parameters():
-                self._hook_variable_gradient_stats(
-                    parameter, 'gradients/' + prefix + name)
+                if parameter.requires_grad:
```
Comment (adrianbg) [inline wandb/wandb_torch.py]: I'm not sure "requires_grad" exists in every PyTorch version. Maybe not in 0.3

Comment (lukas) [reply wandb/wandb_torch.py]: ah good catch

Comment (lukas) [reply wandb/wandb_torch.py]: actually the pytorch 0.3 case works because Tensors are replaced by Variables everywhere

**Pull request merged**